controller/src/controller.rs

```diff
@@ -39,6 +39,14 @@ lazy_static! {
 		HeaderValue::from_str("Basic realm=GrinOwnerAPI").unwrap();
 }
 
+fn check_middleware(
```

Comment:
Middleware goes here (may have to redefine the middleware call to pass in the name of the function as well)